src/style.css

```diff
@@ -1 +1,39 @@
 /* add styles here */
+* {
+  margin: 0;
+}
+
+.stars {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  padding-left: 0;
+}
+
+.stars .stars__star {
+  position: relative;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__star::after {
```

Comment:
you don't need to use pseudo-elements in this task

Reply:
I changed the code.